src/lib/moneyHelpers.ts

```diff
@@ -42,11 +42,14 @@ export const isCurrencySupported = (currency: string): boolean => {
  * Builds price display text (e.g. "$100").
  */
 export const priceDisplayText = (
-  priceCents: number,
+  priceCents: number | [number, number],
   currency: string,
   format: string
 ): string => {
-  return currencyPrefix(currency) + priceAmount(priceCents, currency, format)
+  if (typeof priceCents === "number") {
+    return currencyPrefix(currency) + priceAmount(priceCents, currency, format)
+  }
+  return priceRangeDisplayText(priceCents[0], priceCents[1], currency, format)
```

Comment:
Minor: This might be a little more readable as an `if/else` or even a ternary like `return (typeof PriceCents === 'number') ?  currencyPrefix(currency) + priceAmount(priceCents, currency, format) : priceRangeDisplayText(priceCents[0], priceCents[1], currency, format)` (or maybe not for that one)

Reply:
As priceRangeDisplayText calls priceRangeDisplay I look at number input as at recursion base case instead of simple OR statement.